wagtail/blocks/field_block.py

```diff
@@ -110,8 +113,10 @@ def js_args(self, block):
 
         meta = {
             "label": block.label,
+            "description": block.get_description(),
             "required": block.required,
             "icon": block.meta.icon,
+            "blockDefId": block.definition_prefix,
```

Comment:
Do we want to have a separate method for getting the id, instead of directly accessing `definition_prefix`? (Given that it's not what the variable was originally used for)